marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/run/runFlow.cy.tsx

```diff
@@ -274,6 +274,8 @@ describe("Run Tile tests", () => {
     runPage.deleteStep("map-orders", flowName).click();
     runPage.deleteStepConfirmationMessage("map-orders", flowName);
     cy.findByLabelText("Yes").click();
+    cy.waitForAsyncRequest();
+    cy.wait(5000);
 
     cy.intercept("GET", "/api/jobs/**").as("runResponse");
     runPage.runFlow(flowName);
```

Comment:
@amejiaq Can we add explicit wait instead and try it out. In this case, in runPage.runFlow(flowName) method we can add cy.get(`#runFlow-${flowName}`).should("be.visible", {timeout: 2000}).click({force: true});

Reply:
fixed 🛠